Production/govuk_ios/AppConfig/Providers/AppConfigProviderInterface.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+protocol AppConfigProviderInterface {
+ func fetchAppConfig(filename: String,
+                     completionHandler:
+                     @escaping (Result<AppConfig, AppConfigServiceError>) -> Void)
```

Comment:
Indentation not quite right (only 2 spaces before func + @escaping line should be indented)